### PR TITLE
Update raiden.md

### PR DESCRIPTION
**Pull request by franzihei**
Adds `features` and `getting started` content.

### CODE REVIEW

File: resources/raiden.md [hunk 1]
```diff
@@ -18,8 +18,17 @@ Raiden lets users pay anyone in the network by using a path of connected payment
 
 ## Features
 
--   Feature
--   Feature
+-   Payment channel network for off-chain token transfers
+-   Token networks can be created for any ERC20 compliant token
```
Comment (ligi) [inline resources/raiden.md]: somehow the indentation looks off here - should be the same for all right?

**Review by ligi — CHANGES_REQUESTED** [review]

File: resources/raiden.md (hunk 1 above)
Comment (franzihei) [reply resources/raiden.md]: yeah should be the same!

Comment (ligi) [reply resources/raiden.md]: will u change it?

**Review by ligi — CHANGES_REQUESTED** [review]

File: resources/raiden.md (hunk 1 above)
Comment (franzihei) [reply resources/raiden.md]: should be ok now. :)

**Review by ligi — APPROVED** [review]

**Comment by ligi** [comment]
:+1:

**Pull request merged**